### PR TITLE
Ethereum classic

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

**Comment by aakilfernandes** [comment]
Thanks for this Gav. Could we have classic sync to a different chaindir? A lot of users would likely like to run both chains.

**Comment by gavofyork** [comment]
No sooner said...

File: ethcore/src/client/client.rs [hunk 1]
```diff
@@ -149,9 +149,9 @@ const HISTORY: u64 = 1200;
 const CLIENT_DB_VER_STR: &'static str = "5.3";
 
 /// Get the path for the databases given the root path and information on the databases.
-pub fn get_db_path(path: &Path, pruning: journaldb::Algorithm, genesis_hash: H256) -> PathBuf {
+pub fn get_db_path(path: &Path, pruning: journaldb::Algorithm, genesis_hash: H256, fork_name: Option<&String>) -> PathBuf {
 	let mut dir = path.to_path_buf();
-	dir.push(H64::from(genesis_hash).hex());
+	dir.push(format!("{:?}{}", H64::from(genesis_hash), if let Some(n) = fork_name { format!("-{}", n) } else { "".into() }));
```
Comment (tomusdrw) [inline ethcore/src/client/client.rs]: `fork_name.map(|f| format!("-{}", f)).unwrap_or_default()` ?

Comment (gavofyork) [inline ethcore/src/client/client.rs]: nice

**Pull request merged**